testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -443,7 +457,12 @@ public void testSelectInformationSchemaTables()
     @Test
     public void testSelectInformationSchemaColumns()
     {
-        assertThat(query(format("SELECT column_name FROM information_schema.columns WHERE table_schema = '%s' AND table_name = 'region'", getSession().getSchema().orElseThrow())))
+        if (hasBehavior(SUPPORTS_TIMESERIES_COLUMN)) {
```

Comment:
Time series column isn't common among connectors. Please revert this change in this method.

Reply:
This actually has a default value as false, so only those which would be supported. For pinot, this query fails and so, does the check. Let me know if this needs to be handled in some other way, happy to do so.

Reply:
Please override the test in base Pinot integration smoke test and fix `matches` query.

Reply:
Yes, can look into that. Appreciate the response.